questions-ui.yml

```diff
@@ -0,0 +1 @@
+questions:
```

Comment:
is this ok? or should the file be dropped altogether for the UI to not have a questions tab, given that the policy doesn't have configurable settings?
(@jordojordo ,@kravciak)

Reply:
You should just drop the file and the `kubewarden/questions-ui` property, those won't be needed if there are no settings.

Reply:
Good, I'll remove the file and update the Makefile removing the cli flag to add the questions in the artifacthub-pkg.yml file.